utils/readPkgUp.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+exports.__esModule = true;
```

Comment:
this shouldn't be here; babel adds it.

Reply:
When does babel add it? I was just following the pattern from everything else in `utils/`. If I remove these two lines, the tests fail on some `import` statements.

Reply:
huh, i guess we'll leave it in for now.